### DISCUSSION TITLE
华东师范大学（闵行校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#15] [University]
> 回答来自群友，我仅负责搬运

**宿舍是上床下桌吗**
上床下桌四人间

**教室和宿舍有没有空调**
有空调 图书馆空调冬天太暖和了容易犯困

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
浴室都在宿舍楼1楼 有一栋是混住的，女生洗澡要跑去别的楼

**有早自习、晚自习吗**
有些专业有晚自习

**有晨跑吗**
没有晨跑

**每学期跑步打卡的要求是多少公里，可以骑车吗**
骑就完事了

**寒暑假放多久，每年小学期有多长**
正常寒暑假，在上海这一块假期相对较短你也可以不选小学期

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
疫情前外卖送到楼下

**学校附近有地铁站吗**
地铁在修了

**宿舍楼有洗衣机吗**
宿舍楼六层 隔一层有洗衣机

**校园网怎么样**
在宿舍自己办宽带，电信手机卡套餐39rmb/月，加宽带59rmb/月，100M。裸宽带不知道多少一年。教室和图书馆直接用学工号连校园网，4k不卡

**每天断电断网吗，几点开始断**
不断网不断电

**食堂价格贵吗，会吃出异物吗**
个人感觉还行，就是什么菜都喜欢放糖，不过食堂二楼聚餐相对外面来说很便宜

**洗澡热水供应时间**
看每个宿舍宿管阿姨脸色，一般中午到晚上11点之前都可以，有些楼太晚了一楼得学生会抗议，然后浴室就关的比较早

**校园内可以骑电瓶车吗，电池在哪能充电**
有电动车，但是不知道他们在哪充的电

**Comment by Carriger667** [comment]
宿舍8号楼是男主混住，五六楼男生，一到四女生，男生要去别的楼洗澡

电瓶车充电处在各个宿舍区的地下车库，其余地方最近也新增设了一些充电区，充电区不够的情况有所缓解

食堂小贵，我这种饭桶一天能吃七八十

教学周19周，不过不久后应该会改成17周（编于2025.5.27）

学校附近地铁站有永德路站 紫竹高新区站 以及正在修建的华东师范大学站
